portfolio/src/main/webapp/style.css

```diff
@@ -32,4 +32,7 @@
   color: blanchedalmond;
 }
 
-
+#homelinks img {
+    width: 50px;
+    padding: 10px;
+}
```

Comment:
add newline at the end. github should not be showing a red error icon at the end of the file.